src/components/playerstats/playerstats.js

```diff
@@ -191,6 +191,16 @@ function getDisplayBitrate(bitrate) {
     }
 }
 
+function getDisplayTranscodeFps(session, player) {
+    const mediaSource = playbackManager.currentMediaSource(player) ?? {};
+    const videoStream = (mediaSource.MediaStreams ?? []).find((s) => s.Type === 'Video') ?? {};
```

Comment:
Thank you for the review!
I think it is best to check `videoStream.AverageFrameRate` and return the original string instead of just returning `'-'`

Changing `??` to `||`.